docs-app/app/components/widgets/widget-2/captions.hbs

```diff
@@ -19,31 +19,33 @@
         tabindex="0"
       >
         <h3
-          data-test-field="Music Format"
           class={{local-class
             this.styles
             "music-format"
             (unless CQ.features.large "small-font-size")
           }}
         >
-          <span class={{this.styles.marker}}>
+          <span
+            class={{this.styles.marker}}
+            {{this.colorSvg this.summary.markerColor}}
+          >
```

Comment:
A good example of why a local modifier is useful.

Once [`ember-svg-jar` began to support Glint](https://github.com/ghedamat/ember-svg-jar/pull/226), passing arbitrary named arguments to the `{{svg-jar}}` helper became impossible. For example, in this component, I once passed the `style` property to the helper.

By calling a modifier on the icon container, we can continue to style the svg icon and can separate better API concerns. Such modifier isn't needed anywhere else in this app, so it's good to define the modifier in the component class.